File: 2000~2999/2581.py
M = int(input())
N = int(input())

sum = 0
min = 0

for i in range(M, N+1):

    if i == 1:
        continue

    for j in range(2, i):
        if i % j == 0:
            break
    else:
        if min == 0:
            min = i
        sum += i

if min == 0:
    print(-1)
else:
    print(sum)
    print(min)

File: c/2000~2999/2581.c
#include <stdio.h>
#include <math.h>

int isPrime(int n)
{
	if (n < 2) return 0;
	for (int i = 2; i <= sqrt(n); i++)
	{
		if (n % i == 0) return 0;
	}
	return 1;
}

int main()
{
	int m = 0, n = 0;
	scanf("%d\n%d", &m, &n);

	int t = 0;
	int sum = 0, min = 0;
	for (int i = m; i <= n; i++)
	{
		t = isPrime(i);
		if (t) {
			sum += i;
			if (min == 0) min = i;
		}
	}

	if (min) {
		printf("%d\n%d", sum, min);
	} else	{
		printf("-1");
	}
	return 0;
}
